### PR TITLE
add notes for cadvisor legacy rules

### PR DESCRIPTION
**Pull request by keithhand**
Adding documentation for CAdvisor metrics within legacy environments. Based on [this previous PR
](https://github.com/kubecost/docs/pull/119)
- Related to https://github.com/kubecost/opencost/issues/1271

### CODE REVIEW

File: custom-prom.md [hunk 1]
```diff
@@ -53,6 +53,44 @@ To confirm this job is successfully scraped by Prometheus, you can view the Targ
 
 ![Prometheus Targets](https://raw.githubusercontent.com/kubecost/docs/main/prom-targets.png)
 
+## Cadvisor metric labels
+
+Kubecost uses `container_name` and `pod_name` labels on cadvisor metrics. For clusters running **k8s v1.16+**, the following relabel config creates the expected labels on cadvisor metrics:
+
+```
+  metric_relabel_configs:
```
Comment (AjayTripathy) [inline custom-prom.md:61]: I actually don't think we need any relabeling anymore.

**Comment by AjayTripathy** [comment]
Left some comments there with corrections, though I suggest we try to quickly test these configurations to make sure.

**Review by bstuder99 — APPROVED** [review]

**Comment by srpomeroy** [comment]
@AjayTripathy @keithhand Where are we with this? Any additional work needed to address Ajay's corrections?

**Comment by keithhand** [comment]
@srpomeroy, I never got around to testing with Ajay's suggestions. Let me do that, and I can request another review.

**Review by bstuder99 — DISMISSED** [review]
LGTM

**Comment by Adam-Stack-PM** [comment]
Marking as stale because this PR has had no recent activity. It will be closed in seven days if no further activity occurs.

**Comment by srpomeroy** [comment]
Closing stale issue. Please feel free to re-open when you are ready to revisit.

**Pull request closed without merging**